cli/build.gradle

```diff
@@ -21,4 +23,6 @@ dependencies {
     compile project(':utils')
 
     compile "com.beust:jcommander:$jcommanderVersion"
+    compile "org.jetbrains.kotlin:kotlin-reflect"
```

Comment:
Why no version?

Reply:
Because it it will by default resolve to the same version as the Kotlin plugin for Gradle. I've added that, too, to the commit message.